cdap-app-fabric/src/main/java/co/cask/cdap/gateway/handlers/AppFabricHttpHandler.java

```diff
@@ -1171,8 +1171,7 @@ public void resetCDAP(HttpRequest request, HttpResponder responder) {
       // remove all data in console settings
       consoleSettingsStore.delete();
 
-      // todo: do efficiently and also remove timeseries metrics as well: CDAP-1125
-      appLifecycleHttpHandler.deleteMetrics(Constants.DEFAULT_NAMESPACE, null);
+      namespaceAdmin.deleteMetrics(Constants.DEFAULT_NAMESPACE_ID);
```

Comment:
like @abaranau's comment below, this should go away. deleteMetrics should be part of ``namespaceAdmin.deleteNamespace(Constants.DEFAULT_NAMESPACE_ID)``